src/Support/helpers.php

```diff
@@ -172,7 +172,7 @@ function traceSql()
      */
     function config_path($path = '')
     {
-        return app('path.config').($path ? '/'.$path : $path);
+        return app('path.config').($path ? DIRECTORY_SEPARATOR.$path : $path);
```

Comment:
@mjauvin I wonder if we have to, perhaps, make sure the correct directory separator is also used on the `$path` value provided for this method and the other path helpers as well, perhaps using `str_replace`. While people using Windows might use the correct separator for their own code, there's the potential for plugins to be using this method and not doing the same handling.

Reply:
Also, needs spaces around concatenations:
```suggestion
        return app('path.config') . ($path ? DIRECTORY_SEPARATOR . $path : $path);
```

Reply:
Good idea, I'll submit code to do this, thanks!